### PR TITLE
Update sklearn_train.py

### PR DESCRIPTION
**Pull request by phseidl**
bugfix for single_task_sklearn

### CODE REVIEW

**Review by mliu49 — COMMENTED** [review]
Thanks for the contribution! I added a minor stylistic comment. Otherwise, it looks good to me.

File: chemprop/sklearn_train.py [hunk 1]
```diff
@@ -105,7 +105,11 @@ def single_task_sklearn(model: Union[RandomForestRegressor, RandomForestClassifi
             dataset_type=args.dataset_type,
             logger=logger
         )
-        scores.append(score[0])
+       
+        for metric in metrics:
+            if metric not in scores.keys():
```
Comment (hesther) [inline chemprop/sklearn_train.py:110]: scores.keys() creates a list of dictionary keys, and it is usually preferred to just do `if metric not in scores` instead

**Comment by hesther** [comment]
Thank you very much for contributing to Chemprop, and I apologise for the delayed response. I suggested a minor change (best practice for checking keys in dictionaries) - would you be able to make that change, and also rebase your branch? Thank you!

**Comment by hesther** [comment]
I opened a new pull request so we can make the suggested change and rebase the pr. Am thus closing this PR. Thanks again for contributing to chemprop!

**Pull request closed without merging**